programs/validator-history/src/instructions/update_mev_commission.rs

```diff
@@ -48,16 +47,20 @@ pub struct UpdateMevCommission<'info> {
     pub signer: Signer<'info>,
 }
 
-pub fn handler(ctx: Context<UpdateMevCommission>) -> Result<()> {
+pub fn handler(ctx: Context<UpdateMevCommission>, epoch: u64) -> Result<()> {
     let mut validator_history_account = ctx.accounts.validator_history_account.load_mut()?;
 
     let mut tda_data: &[u8] = &ctx.accounts.tip_distribution_account.try_borrow_data()?;
 
     let tip_distribution_account = TipDistributionAccount::try_deserialize(&mut tda_data)?;
     let mev_commission_bps = tip_distribution_account.validator_commission_bps;
-    let epoch = cast_epoch(Clock::get()?.epoch);
```

Comment:
We need to do a check here and anywhere else `epoch` is passed in that `epoch` is not greater than `Clock::get().epoch`

Reply:
done, the other 2 instructions (update_stake_history and backfill_total_blocks) seem to have the check